Objects/exceptions.c

```diff
@@ -2286,8 +2286,11 @@ MemoryError_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 {
     PyBaseExceptionObject *self;
 
-    if (type != (PyTypeObject *) PyExc_MemoryError)
+    /* If this is a subclass of MemoryError, don't use the freelist
+     * and just return a fresh object */
+    if (type != (PyTypeObject *) PyExc_MemoryError) {
```

Comment:
nitpick: Can you update this line same like line 2323 after you updated this line ;)

Reply:
> nitpick: Can you update this line same like line 2323 after you updated this line ;)
Here it does not make sense because we don't have an instance yet

Reply:
oh, sorry~. Forget my comment.